README.rst

```diff
@@ -436,8 +454,8 @@ This example provides an example of how to extend the base models of
             # your own validation logic here...
             pass
 
-Extending the admin
-~~~~~~~~~~~~~~~~~~~
+Extending admin
```

Comment:
I always see the django admin being referred to as "the admin" or "the django admin", what has prompted you to change this?

Reply:
Extending django-x509
Extending models
Extending ``apps.py``
Extending tests

None of them had "the" in the title, so I removed it from "Extending admin" for consistency.
Should I revert the change?

Reply:
It sounds weird to me, let's ask @2stacks since he's from the US.

Reply:
I would say "Extending the admin model" to be clear but I have seen it both ways depending on the Django experience level of the author.  I think "the admin model" is more clear to new users.